content/en/providers/getting-started/_index.md

```diff
@@ -5,6 +5,17 @@ type: docs
 weight: 10
 ---
 
-{{% pageinfo %}}
-This is a placeholder page.
-{{% /pageinfo %}}
+The pages under this section depict how to join the EGI infrastructure as
+a service providers offering innovative services to the European Research Area.
+
+> Interested in integrating with Check-in? Head to the
```

Comment:
what about IdPs?

Reply:
the integrating was expected to cover both IdP and SP in fact. Maybe we could be more clear and point to the sub sections for both aspects.

Reply:
I've updated to cover this.